bears/general/IndentationBear.py

```diff
@@ -46,8 +47,12 @@ def run(self,
         :param tab_width:
             No. of spaces to insert for indentation.
             Only Applicable if use_spaces is False.
+        :param coalang_dir:
+            Full path of external directory containing the coalang
+            file for language.
         """
-        lang_settings_dict = LanguageDefinition(language)
+        lang_settings_dict = LanguageDefinition(
+                            language, coalang_dir=coalang_dir)
```

Comment:
up to here, all of this is coalang related, right? could that be an own commit similar to https://github.com/coala-analyzer/coala-bears/commit/9542e176071cb64a4f7676f98d09609c64bb3245 maybe?